oak/server/rust/oak_config_serializer/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
This is really part of the SDK, not of the server, so it should be moved under sdk, or as a separate top-level crate.